### ISSUE TITLE
Re-exports in `P5` for most common modules

### ISSUE THREAD

**Issue by thought2** [parenparen/purescript-p5#4]
Currently it may be a bit annoying for the user to import all the needed modules one by one.

What do you think, would it be a good idea to have re-exports in the `P5` module, so that this module is the only one that needs to be imported?

**Comment by parenparen** [comment]
Yes, that sounds good. There shouldn't be name conflicts since the functions are all defined in the same namespace in p5.js.
